src/client/datascience/notebook/vscodeNotebookController.ts

```diff
@@ -88,7 +93,18 @@ export class VSCodeNotebookController implements Disposable {
         this.controller.hasExecutionOrder = true;
 
         // Hook up to see when this NotebookController is selected by the UI
-        this.controller.onDidChangeNotebookAssociation(this.onDidChangeNotebookAssociation, this, this.disposable);
+        this.controller.onDidChangeNotebookAssociation(this.onDidChangeNotebookAssociation, this, this.disposables);
+    }
+
+    public asWebviewUri(localResource: Uri): Uri {
+        const x = this.controller.asWebviewUri(localResource);
```

Comment:
Just return it? Was guessing the x was just for ease of debugging (at least that's what I often do).

Reply:
For debugging, beacuse this wasn't working (ther'es a pending VSCode issue)